### DISCUSSION TITLE
other commit type

### DISCUSSION THREAD

**Discussion by oprogramador** [semantic-release/semantic-release#1864] [Configuration]
How to trigger release on a commit message like `Bump ini from 1.3.5 to 1.3.8 …`?

**Comment by travi** [comment]
More information would be helpful. The message you shared looks like an update from a dependency update service, is that correct? 

If so, the popular services support providing these updates with commits that follow a convention that semantic-release supports. I would recommend consulting the documentation of the service that you are using to determine how to configure it appropriately.

**Reply by oprogramador** [reply]
@travi 

Thanks for your message.

I use dependabot and theoretically it should support custom commit message prefixes but in reality, it doesn't work (https://github.com/dependabot/dependabot-core/issues/3419).

Btw. it would be nice if semantic release could support a not matched commit type so I could configure it to deploy a new patch version for such commits.

**Reply by travi** [reply]
i've had no trouble with dependabot following the convention properly, for example: https://github.com/travi/javascript-scaffolder/pull/650

as mentioned [in the docs](https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates#commit-message), dependabot will normally match your existing convention automatically by analyzing previous commits. if you are wanting to be more explicit, it doesn't look like you are matching the expected config based on your linked issue.

**Reply by oprogramador** [reply]
I cannot find dependabot config in https://github.com/travi/javascript-scaffolder

**Reply by travi** [reply]
> I cannot find dependabot config in [travi/javascript-scaffolder](https://github.com/travi/javascript-scaffolder)

correct, i primarily use renovate for typical updates, so i dont have explicit configuration in that project for dependabot. i do, however, have it enabled in addition to renovate for security updates. the automatic convention matching has worked as expected in every project where i've used it in that way, so i've had no reason to add explicit configuration.

**Reply by oprogramador** [reply]
a workaround is to modify the commit message before merging like here I added `fix: ` before `Merge` https://github.com/oprogramador/sort-any/commits/master
